src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ConvertToString.liquid

```diff
@@ -1,13 +1,12 @@
 ﻿{%              if GenerateNullableReferenceTypes -%}
-[return: System.Diagnostics.CodeAnalysis.NotNullIfNotNull("value")]
-private string? ConvertToString(object? value, System.Globalization.CultureInfo cultureInfo)
+private string ConvertToString(object? value, System.Globalization.CultureInfo cultureInfo)
 {%              else -%}
 private string ConvertToString(object value, System.Globalization.CultureInfo cultureInfo)
 {%              endif -%}
 {
     if (value == null)
     {
-        return null;
+        return "";
```

Comment:
should that be string.Empty?

Reply:
I once was on the "string.Empty" side, but then I heard once that nowadays, the compiler is smart enough to understand that "" and string.Empty are the same.
If there's no perf benefit of using string.Empty, I find "" easier to read.

Reply:
Up to you, we can keep it as is or update it.. for me it's fine either way - it's generated code :-)